apps/heft/src/utilities/Constants.ts

```diff
@@ -3,4 +3,6 @@
 
 export class Constants {
   public static projectHeftFolderName: string = '.heft';
+
+  public static buildCacheFolderName: string = 'build-cache';
```

Comment:
Document the path where this folder can be found. And what it's for. Seems like a somewhat important specification.